docs/build.md

```diff
@@ -48,14 +54,63 @@ This is NOT required when developing for web.
     - In addition, ensure Xcode Command Line Tools are installed using `xcode-select --install`.
   - Expo will require you to configure Xcode Signing. Follow the linked instructions. Error messages in Xcode related to the signing process can be safely ignored when installing on the iOS Simulator; Expo merely requires the profile to exist in order to install the app on the Simulator.
     - Make sure you do have a certificate: open Xcode > Settings > Accounts > (sign-in) > Manage Certificates > + > Apple Development > Done.
+    - For more information on setting up signing certificates: [Expo: Setup Code Signing Certificates in Xcode for Development](https://github.com/expo/fyi/blob/main/setup-xcode-signing.md)
     - If you still encounter issues, try `rm -rf ios` before trying to build again (`yarn ios`)
 - Android: `yarn android`
   - Install "Android Studio"
-    - Make sure you have the Android SDK installed (Android Studio > Tools > Android SDK).
+    - Make sure you have the Android SDK installed (`Android Studio > Settings > Tools > Android SDK`).
```

Comment:
corrected the path based on the latest Android Studio app (for **Mac** -- unconfirmed for **Win** and **Linux** 😬 )